packages/contentful-extension-scripts/scripts/utils/updateExtension.js

```diff
@@ -0,0 +1,10 @@
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+module.exports = async (https = false, port = 1234) => {
```

Comment:
The last thing. I'd remove default values from here cause they already defined in `start.js` and it seems like a duplication of constants for me.

Reply:
Done :)